### PR TITLE
Remove upper case constants

### PR DESCRIPTION
**Pull request by lrhn**

### CODE REVIEW

**Review by zanderso — COMMENTED** [review]
Do you also need to update the pubspec.yaml to indicate the minimum required Dart SDK version?

File: pubspec.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 name: http_io
-version: 0.10.0-dev
+version: 0.10.0
 author: "Dart Team <misc@dartlang.org>"
 homepage: https://github.com/dart-lang/http_io
 description: HTTP Client and Server APIs.
 environment:
-  sdk: ">=1.24.0 <2.0.0"
+  sdk: ">=2.0.0-dev.17.0 <2.0.0"
```
Comment (zanderso) [inline pubspec.yaml:7]: Oh, sorry, you did it here. Thanks!

**Review by zanderso — APPROVED** [review]

File: pubspec.yaml (hunk 1 above)
Comment (zanderso) [reply pubspec.yaml:7]: But we should probably also inc the package version and republish, I guess. Is that right, @kevmoo ?

Comment (lrhn) [reply pubspec.yaml:7]: The package version is incremented from 0.10.0-dev to 0.10.0 so it can be published when landed.

**Comment by lrhn** [comment]
I have merged the change, so please go ahead and publish it.
Cheers

**Comment by kevmoo** [comment]
This isn't published yet AFAIK

**Pull request merged**